src/Kunstmaan/NodeSearchBundle/Configuration/NodePagesConfiguration.php

```diff
@@ -65,9 +63,6 @@ class NodePagesConfiguration implements SearchConfigurationInterface
     /** @var LoggerInterface */
     protected $logger = null;
 
-    /** @var IndexablePagePartsService */
-    protected $indexablePagePartsService;
```

Comment:
We can't remove this property now as it's protected. Add a `@deprecated` annotation so we can remove it in 6, together with the other deprecated methods